.docker/api-gateway-nginx.conf

```diff
@@ -9,6 +9,7 @@ server {
         add_header Content-Type "text/plain; charset=utf-8";
         add_header Expires 0;
         return 200 "pong";
+        access_log off;
```

Comment:
Assume the order doesn't matter?

Reply:
It doesn't for these rules afaik, any preference?

Reply:
Gut reaction is for 'settings' to come first, then content (with `return` at the end). 🤷‍♂️ though.